packages/hardware-ledger/src/LedgerKeyAgent.ts

```diff
@@ -40,6 +40,16 @@ const LedgerConnection = (_LedgerConnection as any).default
   : _LedgerConnection;
 type LedgerConnection = _LedgerConnection;
 
+const isDeviceAlreadyOpenError = (error: unknown) => {
+  if (typeof error !== 'object') return false;
```

Comment:
`null` is an object. I think might cause issues on the following line. Can you add some unit tests for this function?